packages/react-ai/src/types/AIConversation.ts

```diff
@@ -0,0 +1,23 @@
+export interface ImageContent {
+  format: 'png' | 'jpeg' | 'gif' | 'webp';
+  bytes: ArrayBuffer;
+}
+
+interface ImageContentBlock {
+  type: 'image';
+  value: ImageContent;
+}
+
+export interface TextContent {
+  type: 'text';
+  value: string;
+}
+
+export type Content = ImageContentBlock | TextContent;
+
+export interface ConversationMessage {
+  id: string;
+  content: Content;
```

Comment:
Should this be `Content[]`?

Reply:
Oh actually I think these are unused since im importing directly from the data-schema package, will remove